From dc0781004c9634bdbc0ce51769d4de1f5da06360 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Tue, 2 Dec 2014 10:58:10 +0800
Subject: [PATCH 1/3] atomisp2: Remove symbol links for atomisp_driver

Currently atomisp2 driver uses symbol links which links to
atomisp common driver folder for different platform build.
But this causes conflicts since GMIN and IMIN kernel build
directly output the object files to the source folder.
This patch removes the original symbol links for atomisp_driver.

Change-Id: I603d71fc29c2cb9c747351a0150f8c8c5adc1770
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-9315
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://android.intel.com:443/307204
---
 .../atomisp2/css2400b0_v21_build/atomisp_driver    |    1 -
 .../css2401a0_legacy_v21_build/atomisp_driver      |    1 -
 .../atomisp2/css2401a0_v21_build/atomisp_driver    |    1 -
 3 files changed, 3 deletions(-)
 delete mode 120000 drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/atomisp_driver
 delete mode 120000 drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_legacy_v21_build/atomisp_driver
 delete mode 120000 drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_v21_build/atomisp_driver

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/atomisp_driver b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/atomisp_driver
deleted file mode 120000
index 0bd2c7d..0000000
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21_build/atomisp_driver
+++ /dev/null
@@ -1 +0,0 @@
-../atomisp_driver
\ No newline at end of file
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_legacy_v21_build/atomisp_driver b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_legacy_v21_build/atomisp_driver
deleted file mode 120000
index 0bd2c7d..0000000
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_legacy_v21_build/atomisp_driver
+++ /dev/null
@@ -1 +0,0 @@
-../atomisp_driver
\ No newline at end of file
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_v21_build/atomisp_driver b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_v21_build/atomisp_driver
deleted file mode 120000
index 0bd2c7d..0000000
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2401a0_v21_build/atomisp_driver
+++ /dev/null
@@ -1 +0,0 @@
-../atomisp_driver
\ No newline at end of file
-- 
1.7.9.5

